src/ui/public/management/section.js

```diff
@@ -49,10 +50,18 @@ export class ManagementSection {
     this.url = '';
 
     assign(this, options);
+
+    if (this.parent) {
+      this.capabilityId = camelCase(`${this.parent.id} ${this.id}`);
```

Comment:
**problem** - the capability ids right now are a camel-cased concatenation of the section id and the link id. So the "Advanced Settings" link with an id of `settings` within the `kibana` section becomes `kibanaSettings`. This is neither intuitive or grep-able when searching for usages. We need the combination of section id and link id because it's possible to have identical link identifiers within unique sections.

**proposal 1** - we could use dot notation instead, which would make it a little clearer. The example above would become `kibana.settings`. The only downside here is that the capability declaration in `src/legacy/core_plugins/kibana/index.js` would look like this, which we tried to avoid in the past:

```
            management: {
              /*
               * Management settings correspond to management section ids, and should not be changed
               * without also updating those sections.
               */
              'kibana.settings': true,
              'kibana.indices': true,
            }
```

**proposal 2** - we could turn `management` into a nested object, which follows the management hierarchy. The downside with this is that it makes merging this object rather complicated, as plugins can register their management links within any management section, not just the ones they create. This breaks precedent that we've set for other UI Capability buckets. 
```
management: {
  kibana: {
    settings: true,
    indices: true,
  }
}
```

Reply:
I think I prefer "proposal 1". We're using the ":" for the complex navlinks, perhaps doing so here as well would be "consistent"? I wonder if we could introduce a facility similar to what we're doing for "actions" which would let us do `managementUICapabilities.get('section', 'id)` here and where the inject Default Vars are specified?

Reply:
Can I take it all back? "proposal 2" is growing on me, because it'd make the string concatenation only relevant when doing the actions building for serialization in ES, which seems appropriate. How gnarly do you feel that the management merging would end up becoming? We primarily want to prevent anyone redefining a key that isn't there before, right?

Reply:
Implemented proposal 2 via https://github.com/elastic/kibana/pull/27055/commits/262ed93872b4b7afa5e28fb9aeb37ce44545bbd7